<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="LWTargetFiles">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>111</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>0</y>
      <width>77</width>
      <height>197</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="BTNCompile">
       <property name="text">
        <string>Compile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BTNExecute">
       <property name="text">
        <string>Execute</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="consoleTextEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>440</y>
      <width>569</width>
      <height>196</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>541</width>
      <height>421</height>
     </rect>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>970</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Temporary_File"/>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionShow_opened_files"/>
    <addaction name="actionShow_console_output"/>
    <addaction name="actionEdit_execution_map"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionCompile"/>
    <addaction name="actionExecute"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindows"/>
   <addaction name="menuActions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionNew_Temporary_File">
   <property name="text">
    <string>New Temporary File</string>
   </property>
  </action>
  <action name="actionShow_opened_files">
   <property name="text">
    <string>Show opened files</string>
   </property>
  </action>
  <action name="actionShow_console_output">
   <property name="text">
    <string>Show console output</string>
   </property>
  </action>
  <action name="actionCompile">
   <property name="text">
    <string>Compile - CTRL + E</string>
   </property>
  </action>
  <action name="actionExecute">
   <property name="text">
    <string>Execute - CTRL + R</string>
   </property>
  </action>
  <action name="actionEdit_execution_map">
   <property name="text">
    <string>Edit execution map</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
